message-me/app/controllers/sessions_controller.rb

```diff
@@ -1,4 +1,6 @@
 class SessionsController < ApplicationController
+  before_action :logged_in_redirect, only: [:new, :create]
```

Comment:
```suggestion
  before_action :logged_in_redirect, if: :logged_in?, only: [:new, :create]
```

Please test the above. I not sure if my syntax correct

Reply:
yep it works